test/examples/correct/testudinata_phylonym.json

```diff
@@ -0,0 +1,125 @@
+{
+    "@context": "../../../docs/context/development/phyx.json",
```

Comment:
Remind me why we are using the development version here?

Reply:
The purpose of these example files and the test suite is to test the development version of the Phyx context file -- the code that generates the phyloreferences and the context file need to be tied very closely together during development, so that changes in property names or how they are used are interpreted correctly using the context file to generate valid OWL. The previous version of the Phyx context file will not work on currently generated phyloreferences, because it is out of date in terms of both property names and IRI mappings.

Leaving it pointing to `development` also means that we don't need to change the example files every time we release a new Phyx context file: instead, we can just copy the development file (and the JSON schema) into a new directory (e.g. `v0.3.0` or `v1.0.0`), and then continue development in the `development` directory.